node/src/chain_spec.rs

```diff
@@ -173,7 +173,62 @@ pub fn remote_testnet_config() -> Result<ChainSpec, String> {
 		Some("Golden Gate"),
 		None,
 		// Properties
-		properties(),
+		properties("GGXT"),
+		// Extensions
+		None,
+	))
+}
+
+#[cfg(not(feature = "testnet"))]
+pub fn sydney_testnet_config() -> Result<ChainSpec, String> {
+	use sc_telemetry::TelemetryEndpoints;
+
+	let wasm_binary = WASM_BINARY.ok_or_else(|| "Runtime wasm not available".to_string())?;
+
+	Ok(ChainSpec::from_genesis(
+		"GGX Sydney Testnet",
+		"GGX",
+		ChainType::Live,
+		move || {
+			testnet_genesis(
+				wasm_binary,
+				// Sudo account
+				sr25519::Public::from_ss58check("5EHkPQgHPKLT4XTEkZcVWpwvLziBS3Qf2oUg94YAk79YVFdw")
+					.unwrap()
+					.into_account()
+					.into(),
+				// Pre-funded accounts
+				vec![sr25519::Public::from_ss58check(
+					"5EHkPQgHPKLT4XTEkZcVWpwvLziBS3Qf2oUg94YAk79YVFdw",
```

Comment:
Regarding pre-funded accounts, I'll provide some info a bit later, when we set up multisig wallets.